docs/Users_Guide/glossary.rst

```diff
@@ -13319,6 +13269,772 @@ METplus Configuration Glossary
 
      | *Used by:* WaveletStat
 
```

Comment:
Please edit the entry for `MODEL` to add `PairStat` to the list of wrappers that use that entry.